docs/developer-docs/latest/developer-resources/database-apis-reference/rest/populating-fields.md

```diff
@@ -0,0 +1,497 @@
+---
+title: Population for REST API - Strapi Developer Docs
+description: Use Strapi's REST API to populate or select certain fields.
+sidebarDepth: 3
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest/populating-fields.html
+---
+
+# REST API: Population & Field Selection
+
+The [REST API](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md) by default does not populate any relations, media fields, components, or dynamic zones. It will return all fields for the model and while populating.
+
+## Field selection
+
+Queries can accept a `fields` parameter to select only some fields. By default Strapi will return all normal fields and field selection will not work on relational, media, component, or dynamic zone fields, to see how to properly populate these refer to the [population documentation](#population).
```

Comment:
Not sure we should use "normal". We should be more specific. What's a 'normal` field?
Suggestion:
"By default, only the following types of fields are returned: [ insert types that are returned here].

Fields selection does not work on relational, media, component, or dynamic zone fields. Use [population](#population) instead.